spec/models/recipe_spec.rb

```diff
@@ -0,0 +1,51 @@
+require_relative '../rails_helper'
+
+RSpec.describe Recipe, type: :model do
+  user = User.create!(name: 'Tomas', email: 'tomas@email.com', password: '123456')
+  subject do
+    user.recipes.new(name: 'Apple Cake', preparation_time: '10m',
+                     cooking_time: '10m', description: 'This is a easy recipe to follow', public: false)
+  end
+
+  before(:example) { subject.save }
+
+  it 'Validation of the new recipe' do
+    expect(subject).to be_valid
+  end
+
+  context 'Testing name argument' do
+    it 'Name must be present' do
+      subject.name = nil
+      expect(subject).to_not be_valid
+    end
+
+    it 'Name must be less than 30 char' do
+      subject.name = 'tomasrichradlisonjosedelasantisimatrinidadpontepalaciosyblanco'
+      expect(subject).to_not be_valid
+    end
+  end
+
+  context 'Testing preparation time argument' do
+    it 'Should be lesser than 5 char' do
+      subject.preparation_time = '10 minutes'
+      expect(subject).to_not be_valid
+    end
+
+    it 'Should not be nil' do
+      subject.preparation_time = nil
+      expect(subject).to_not be_valid
+    end
+  end
+
+  context 'Testing coooking time argument' do
+    it 'Should be lesser than 5 char' do
+      subject.cooking_time = '10 minutes'
+      expect(subject).to_not be_valid
+    end
+
+    it 'Should not be nil' do
+      subject.cooking_time = nil
+      expect(subject).to_not be_valid
+    end
+  end
+end
```

Comment:
- Please note that some of your tests are not passing. Kindly fix them 😉 
![Screenshot from 2022-11-11 00-43-31](https://user-images.githubusercontent.com/44624138/201231294-035b07c7-c5fa-4ec0-94df-59278e58b1c6.png)

Reply:
![Screenshot from 2022-11-11 10-43-27](https://user-images.githubusercontent.com/102382778/201363841-8f92bb5b-2193-41fd-a48b-3dd7e9daacee.png)
Thank you for your observation we already fix them all.